internal/pkg/service/buffer/api/service/service.go

```diff
@@ -397,13 +403,27 @@ func (*service) Import(d dependencies.ForPublicRequest, payload *buffer.ImportPa
 		}
 
 		// nolint:godox
-		// TODO get sliceID
+		// TODO get sliceID and fileID + use in stats.Notify
 
 		record := key.NewRecordKey(e.ProjectID, e.ReceiverID, e.ExportID, receivedAt, receivedAt)
 		err = str.CreateRecord(ctx, record, csv)
 		if err != nil {
 			errs.AppendWithPrefixf(err, `failed to create record for export "%s"`, e.ExportID)
 		}
+
+		key := key.NewSliceStatsKey(
+			e.ProjectID,
+			e.ReceiverID,
+			e.ExportID,
+			receivedAt,
```

Comment:
@michaljurecko ještě tady jedna věc. Ten klíč pro stats obsahuje nejen `sliceId` ale i `fileId`, jak se k tomu tady dostanu? Mám to vůbec řešit?

Reply:
Sorry, až teď jsem si všiml, že to je okomentované v Jira issue.

Reply:
V pohode, napadlo mi to az po calle.